### DISCUSSION TITLE
Is Datadog tracing supported by envelop?

### DISCUSSION THREAD

**Discussion by riccardogiorato** [dotansimha/envelop#466] [Q&A] [answered]
Is Datadog tracing supported by envelop?

In the "announcement" article here is missing a link to Datadog plugin: https://the-guild.dev/blog/introducing-envelop

Is this a "work-in-progress" plugin? Is there a related PR?

**Comment by dotansimha** [comment]
Hi @riccardogiorato ! Sorry for the late reply. 
We are currently working on a DataDog plugin. Hope to have a PR soon.

In the meantime, ou can use `opentelemetry` plugin with DataDog (see https://docs.datadoghq.com/tracing/setup_overview/open_standards/), and you can also use `prometheus` plugin with DataDog (https://docs.datadoghq.com/integrations/prometheus/). 

The new plugin will implement a tracing solution based on `dd-trace`, and will probably be very similar, we are still experimenting and trying to find the ideal way to integrate.

**Reply by riccardogiorato** [reply]
Would love to see the implementation done with dd-trace, cause it would make it the easiest to work with datadog cause opentelemetry data sent to datadog needs mulitple modifications currently to be fully supported.

**Comment by kamilkisiela** [comment, marked as answer]
https://github.com/dotansimha/envelop/pull/533 currently in progress

**Reply by dotansimha** [reply]
Will be available soon as `@envelop/statsd@0.1.0` :)
